examples/flutter_gallery/test/demo/material/menu_demo_test.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
this can be `2020` instead of `2014`.

Reply:
Many files in `examples/flutter_gallery` use `2014`, although they were created in 2017. What is the rule we should follow for the copyright header?

Reply:
These should be 2014